doc/code_snippets/snippets/connectors/instances.enabled/sample_db/myapp.lua

```diff
@@ -0,0 +1,24 @@
+-- Create a space --
+box.schema.space.create('bands')
+
+-- Specify field names and types --
+box.space.bands:format({
+    { name = 'id', type = 'unsigned' },
+    { name = 'band_name', type = 'string' },
+    { name = 'year', type = 'unsigned' }
+})
+
+-- Create indexes --
+box.space.bands:create_index('primary', { parts = { 'id' } })
+box.space.bands:create_index('band', { parts = { 'band_name' } })
+box.space.bands:create_index('year', { parts = { { 'year' } }, unique = false })
+box.space.bands:create_index('year_band', { parts = { { 'year' }, { 'band_name' } } })
```

Comment:
The `'year'` index looks redundant, because requests can use the `'year_band'` index with a partial key (`{2024}` for example).